app.js

```diff
@@ -1,44 +1,59 @@
-const express = require('express');
-const path = require('path');
-const bodyParser = require('body-parser');
-const cors = require('cors');
-const mongo = require('mongoose');
-const dbConfig = require('./config/database');
-const portalControl = require('./controllers/portal-controller');
+let express = require('express');
+let path = require('path');
+let bodyParser = require('body-parser');
+let cors = require('cors');
+let mongo = require('mongoose');
+let morgan = require("morgan");
+let config = require('config');
+const projects = require('./controllers/projects-controller');
 
 // initialize app variable
-const app = express();
+let app = express();
 
 // declare port
 const port = 3000;
 
 // connect to database
-mongo.connect(dbConfig.database);
+mongo.connect(config.dbHost);
+// error logging
+let db = mongo.connection;
+db.on('error', console.error.bind(console,'connection error:'));
+
+// don't show log when in test mode
+if(config.util.getEnv('NODE_ENV') !== 'test') {
+    // use morgan to log at command line
+    app.use(morgan('combined')); //'combined' mode creates Apache-style logs
+}
 
 // middleware
 app.use(cors());
 
 // middleware for bodypasrsing using json and urlencoding
 app.use(bodyParser.urlencoded({extended:true}));
 app.use(bodyParser.json());
+app.use(bodyParser.json({type:'application/json'}));
+app.use(bodyParser.text());
 
 // declare public folder
 app.use(express.static(path.join(__dirname,'public')));
 
 
-// routes
-// for testing purposes during setup
-app.get('/hello', (req,res) => {
-   res.send("Hello, world!");
+app.get('/', (req,res) => {
+   res.send("Welcome to Projects Portal!");
 });
 
-app.use('/',portalControl);
-// KEEP the following: postman instructions depend upon it
-// TODO change postman instructions
-// app.use('/portal',portalControl);
+app.route('/view').get(projects.getAllProjects);
```

Comment:
Guys:@redemma71 @andrewoconnell89 

With this implementation "/" corresponds to the "projects" module.
Since we have other models such as User, Comments etc in the pipeline we should probably change the route to:
/project/add
/project/view
/project/view/:id

and subsequently for users:
/user/view
/user/add
/user/delete

Do you agree?